app/controllers/groups_controller.rb

```diff
@@ -117,6 +117,58 @@ def index
                                    .order(:id)
   end
 
+  def assign_scans
+    @assignment = Assignment.find(params[:assignment_id])
+    @current_grouping = Grouping.get_assign_scans_grouping(@assignment,
+                                                      @current_user)
+    if @current_grouping == nil
+      redirect_to(:back)
+    end
+  end
+
+  def get_names
+    names = Student.select(:id, 'CONCAT(first_name,\' \',last_name) AS label', 'CONCAT(first_name,\' \',last_name) AS value')
+              .where('lower(first_name) like ? OR lower(last_name) like ?', "#{params[:term].downcase}%", "#{params[:term].downcase}%")
+    render json: names
+  end
+
+  def assign_student_and_next
+    if params[:s_id].present?
+      @student = Student.find(params[:s_id])
```

Comment:
Do not use instance variables (ones that start with `@`) unless absolutely necessary. They aren't here.